src/dataviews/histogram-dataview-model.js

```diff
@@ -48,12 +49,29 @@ module.exports = DataviewModelBase.extend({
     // BBox should only be included until after the first fetch, since we want to get the range of the full dataset
     this.once('change:data', function () {
       this.set('submitBBox', true);
+
+      var data = this.getData();
+      if (data && data.length > 0) {
+        this.set({
```

Comment:
be careful with calling `set` on the initialize method, if you want to do this sometimes is better to change attributes directly. The reason is Backbone keeps changes for later so if you do `set('other_attr', 'test')` later you will get `change:bins` triggered